content/variables/git.md

```diff
@@ -51,4 +51,21 @@ The `GitInfo` object contains the following fields:
 `.Subject`
 : commit message subject (e.g., `tpl: Add custom index function`)
 
+### Debugging `.GitInfo`
```

Comment:
Please restrict the commit to what the commit message says. I like your Lastmod change below, not the debug part here, which has no place on this page.

Reply:
OK, makes sense.. I will remove it from this commit. Let me know the correct location for this.. I just wanted to make a point that `range` cannot be used. I had made that mistake and this this showed up on discourse ( https://discourse.gohugo.io/t/iterate-over-gitinfo/8472 ). So I thought this should be useful to others too.